backend/main/server/controllers.go

```diff
@@ -894,6 +901,57 @@ func (a *App) createCommunityUser(w http.ResponseWriter, r *http.Request) {
 	respondWithJSON(w, http.StatusCreated, "OK")
 }
 
+func (a *App) getUser(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	addr := vars["addr"]
+
+	var user models.User
+	if err := user.GetUser(a.DB, addr); err != nil {
+		log.Error().Err(err).Msg("Error user not found")
+		respondWithError(w, errUserNotFound)
+		return
+	}
+
+	respondWithJSON(w, http.StatusOK, user)
+}
+
+func (a *App) createUser(w http.ResponseWriter, r *http.Request) {
+	payload := models.User{}
+
+	if err := validatePayload(r.Body, &payload); err != nil {
+		log.Error().Err(err).Msg("Error validating payload")
+		respondWithError(w, errIncompleteRequest)
+		return
+	}
+
```

Comment:
we should definitely require/validate a signature to create & update a user, otherwise the API is wide open.

you can ignore the `Voucher` case, because Dapper Wallet has finally implemented user signing, so that will most likely be going out with this release as well.